res/test_generate_update_metadata.py

```
@@ -0,0 +1,270 @@
+        private_key = ed25519.Ed25519PrivateKey.generate()
+        seed = private_key.private_bytes(
+            encoding=serialization.Encoding.Raw,
+            format=serialization.PrivateFormat.Raw,
+            encryption_algorithm=serialization.NoEncryption(),
+        )
+        public_key = private_key.public_key().public_bytes(
+            encoding=serialization.Encoding.Raw,
+            format=serialization.PublicFormat.Raw,
+        )
+        return base64.b64encode(seed).decode("ascii"), base64.b64encode(public_key).decode("ascii")
+    def run_script(self, *args, env=None, check=True):
```

Ruff (0.16.1)

[warning] Missing type annotation for `*args`

(ANN002)